easybuild/easyconfigs/s/spoa/spoa-4.0.7-GCC-10.2.0.eb

```diff
@@ -14,7 +14,7 @@ source_urls = ['https://github.com/rvaser/spoa/releases/download/%(version)s/']
 sources = ['spoa-v%(version)s.tar.gz']
 checksums = ['f7a8cd039e4aabea1481e123387fedaa8f87e55418a3649408a615f5c6b1b9a4']
 
-builddependencies = [('CMake', '3.18.4')]
+dependencies = [('CMake', '3.18.4')]
```

Comment:
Again, this looks wrong to me... In what sense is `CMake` a runtime dependency here?